From 89b520675f3a40f7a6396a79fd1105a3aaf9b8ec Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Tue, 1 Jul 2014 14:09:51 -0400
Subject: [PATCH] Move rabbitmq-server cluster port

This is a simplified version of the upstream patch
https://review.openstack.org/#/c/91704/11

This patch is much simpler for that one in that it
doesn't also require the sysctl element to be backported.

Change-Id: Ib58dd7d3ecfada47a5f4e7ef2745618ab718c35c
---
 .../rabbitmq-server/os-config-applier/etc/rabbitmq/rabbitmq.config    | 4 ++--
 .../os-refresh-config/pre-configure.d/98-rabbitmq-iptables            | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/elements/rabbitmq-server/os-config-applier/etc/rabbitmq/rabbitmq.config b/elements/rabbitmq-server/os-config-applier/etc/rabbitmq/rabbitmq.config
index 840569e..dc9121e 100644
--- a/elements/rabbitmq-server/os-config-applier/etc/rabbitmq/rabbitmq.config
+++ b/elements/rabbitmq-server/os-config-applier/etc/rabbitmq/rabbitmq.config
@@ -1,8 +1,8 @@
 [
   {kernel,
     [
-      {inet_dist_listen_min, 5535},
-      {inet_dist_listen_max, 5535}
+      {inet_dist_listen_min, 61000},
+      {inet_dist_listen_max, 61000}
     ]
   }
 ].
diff --git a/elements/rabbitmq-server/os-refresh-config/pre-configure.d/98-rabbitmq-iptables b/elements/rabbitmq-server/os-refresh-config/pre-configure.d/98-rabbitmq-iptables
index 4b52909..fca51fe 100755
--- a/elements/rabbitmq-server/os-refresh-config/pre-configure.d/98-rabbitmq-iptables
+++ b/elements/rabbitmq-server/os-refresh-config/pre-configure.d/98-rabbitmq-iptables
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -eu
 
-add-rule INPUT -p tcp -m multiport --dports 4369,5535 -j ACCEPT
+add-rule INPUT -p tcp -m multiport --dports 4369,61000 -j ACCEPT
-- 
1.9.3

